Calibration/IsolatedParticles/src/eECALMatrix.cc

```diff
@@ -160,7 +164,96 @@ namespace spr{
       }
     }
 #ifdef EDM_ML_DEBUG
-    if (debug) std::cout << "energyECAL: energySum = " << energySum << " flag = " << flag << std::endl;
+    if (debug) std::cout << "energyECAL: energySum = " << energySum 
+			 << " flag = " << flag << std::endl;
+#endif
+    return std::pair<double,bool>(energySum,flag);
+  }
+
+  std::pair<double,bool> eECALmatrix(const HcalDetId& detId, 
+				     edm::Handle<EcalRecHitCollection>& hitsEB,
+				     edm::Handle<EcalRecHitCollection>& hitsEE,
+				     const CaloGeometry* geo, 
+				     const CaloTowerConstituentsMap* ctmap,
+				     const EcalSeverityLevelAlgo* sevlv,
+				     double ebThr, double eeThr, double tMin, 
+				     double tMax, bool debug) {
+
+    CaloTowerDetId tower = ctmap->towerOf(detId);
+    std::vector<DetId> ids = ctmap->constituentsOf(tower);
+#ifdef EDM_ML_DEBUG
+    if (debug) {
+      std::cout << "eECALmatrix: " << detId << " belongs to " << tower 
+		<< " which has " << ids.size() << " constituents" << std::endl;
+      for (unsigned int i=0; i<ids.size(); ++i) {
+	std::cout << "[" << i << "] " << std::hex << ids[i].rawId() <<std::dec;
+	if        (ids[i].det()==DetId::Ecal && ids[i].subdetId()==EcalBarrel){
+	  std::cout << " " << EBDetId(ids[i]) << std::endl;
+	} else if (ids[i].det()==DetId::Ecal && ids[i].subdetId()==EcalEndcap){
+	  std::cout << " " << EEDetId(ids[i]) << std::endl;
+	} else if (ids[i].det()==DetId::Ecal && ids[i].subdetId()==EcalPreshower) {
+	  std::cout << " " << ESDetId(ids[i]) << std::endl;
+	} else if (ids[i].det()==DetId::Hcal) {
+	  std::cout << " " << HcalDetId(ids[i]) << std::endl;
+	} else {
+	  std::cout << std::endl;
+	}
+      }
+    }
+#endif
+
+    bool   flag(true);
+    double energySum(0);
+    const EcalRecHitCollection* recHitsEB = (hitsEB.isValid()) ? 
+      hitsEB.product() : nullptr;
+    for (const auto& id : ids) {
+      if (id.det() == DetId::Ecal) {
+	bool ok(true);
+	std::vector<EcalRecHitCollection::const_iterator> hits;
+        if (id.subdetId()==EcalBarrel) {
+          spr::findHit(hitsEB,id,hits,debug);
+	  ok  = (sevlv->severityLevel(id,(*recHitsEB)) != EcalSeverityLevel::kWeird);
+        } else if (id.subdetId()==EcalEndcap) {
+          spr::findHit(hitsEE,id,hits,debug);
+        }
+#ifdef EDM_ML_DEBUG
+        if (debug) std::cout << "Xtal 0x" << std::hex << id() <<std::dec;
+#endif
+        double ener(0);
```

Comment:
and here (and just below)

Reply:
Don't understand what you mean for this

Reply:
I mean: "please make the indentation consistent in this stanza of code"